packages/@aws-cdk/aws-appmesh/test/integ.mesh.ts

```diff
@@ -23,11 +23,21 @@ const router = mesh.addVirtualRouter('router', {
   ],
 });
 
-const virtualService = new appmesh.VirtualService(stack, 'service', {
+const virtualService1 = new appmesh.VirtualService(stack, 'service1', {
```

Comment:
Let's leave this ID as-is, and just start adding `'service2'` below. We should minimize these changes as much as possible.